config/company_config.dist.yml

```
@@ -1,7 +1,14 @@
 sign_cipher: 'XYZ' # add your sign cipher
 test_devices:
     # add your test device numbers
-    - '1234.5X.67890'
+    '1234.5X.67890'
```

Didn't you forget something?

```suggestion
    '1234.5X.67890':
```